sql/firefox_accounts_exact_mau28_by_dimensions_v1.sql

```diff
@@ -0,0 +1,18 @@
+SELECT
```

Comment:
Instead of a comment at the bottom, we could do this as DDL
```suggestion
CREATE OR REPLACE VIEW
  `firefox_accounts_exact_mau28_by_dimensions_v1` AS
SELECT
```

Reply:
Love it.